nntrainer/layers/multi_head_attention_layer.h

```diff
@@ -161,7 +162,12 @@ class MultiHeadAttentionLayer : public LayerImpl {
 
       auto sin = new std::vector<std::vector<float>>();
       sin->assign(seq_len, std::vector<float>(dim, 0));
-
+#ifdef USE_NEON
+      for (unsigned int i = 0; i < seq_len; ++i) {
+        calc_trigonometric_vals_dup(half_, freqs.data(), (*cos)[i].data(),
+                                    (*sin)[i].data(), i);
+      }
+#else
       for (unsigned int i = 0; i < seq_len; ++i) {
```

Comment:
```suggestion
      for (unsigned int i = 0; i < seq_len; ++i) {
#ifdef USE_NEON
        calc_trigonometric_vals_dup(half_, freqs.data(), (*cos)[i].data(),
                                    (*sin)[i].data(), i);
#else
```

this could reduce code

Reply:
Nice catch!
Will refactor immediately